interface/patient_file/front_payment.php

```diff
@@ -10,8 +10,6 @@
  * @copyright Copyright (c) 2017 Brady Miller <brady.g.miller@gmail.com>
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
```

Comment:
This script is reverting very large amount of previous code changes. This is likely because it's a file with lots of relatively recent changes. See here for the commit history and detailed changes of this script: https://github.com/zbig01/openemr/commits/f711374194f0b3d239cdfa2704c863ebab34529b/interface/patient_file/front_payment.php
(in this case, I think it may actually be easier to get the script from current official master branch and then make your changes again to this script)

Reply:
TODO; I will re-analyze this.

Reply:
hi @zbig01 , There are still a lot of missing prior changes in this script. See my comment above regarding this.